ui/main.qml

```diff
@@ -25,8 +25,8 @@ ApplicationWindow {
     color: Style.current.background
     title: {
         // Set application settings
-        //% "Nim Status Client"
-        Qt.application.name = qsTrId("nim-status-client")
+        //% "Status Desktop"
+        Qt.application.name = qsTrId("status-desktop")
```

Comment:
This needs to match the message-id in the .ts files: <message id="nim-status-client">

Reply:
Hey. When i change this string to the id nim-status-client, the running application reverts back to "Nim Status Desktop" even though the source for the id `nim-status-client` (in all the translation files) is in fact `Status Desktop`. I've decided to add a new commit that rather changes all the message ids to `status-desktop` for consistency sake.